content/en/news/releases/1.13.x/announcing-1.13/change-notes/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Istio 1.13 Change Notes
+linktitle: 1.13.0
+subtitle: Minor Release
+description: Istio 1.13.0 change notes.
+publishdate: 2021-02-08
```

Comment:
One other thing, passed on looking at past Change Notes, is we usually group the same types of changes together under each heading, and they seem to be in an Improved, Updated, Added, Fixed, Promoted order.

Reply:
Looks like we still want to reorder these into the groups.